.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# How to contribute
+
+We are really glad you are reading this contribution guide. This means you care about the quality of your contributions.
+
+## Submitting patches via Github
+
+### Prerequisite for submitting patches/PR's
+
+1. Basic knowledge about how to us Git(hub)
+  * Properly setting up git with `git config`
+  * Know how to create a commit (for details see below).
+  * Know how to work with git history (rebasing your commits).
+  * Know how to create Pull Requests
+2. Have access to an Alpine Linux [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development).
+3. Please __do not__ submit PR's via Github's web interface. You should have a working [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development) available and submit your commits from the git interface.
+
+### Creating a Pull Request (PR)
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) our alpinelinux aports repository.
+2. Clone your copy of aports `git clone https://github.com/username/aports`
+3. Create a feature branch `git checkout -b my_new_feature`
+4. Make your desired changes and commit them with a correct commit message.
+  * Begin the commit message with a single short (less than 78 character) line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correct formated summary lines are:
```

Comment:
Actually it should be less than 69 characters. GitHub truncates any subject line longer than 69 characters with an ellipsis. I should also note that this is not “invention” of GitHub, it originates in git itself.

BTW The body of commit message should be wrapped at 72 characters, but that’s not so important as 69 characters in title.